linkerd2-proxy/src/main.rs

```diff
@@ -21,9 +21,9 @@ fn main() {
         }
     };
 
-    tokio_compat::runtime::current_thread::Runtime::new()
+    tokio::runtime::Runtime::new()
         .expect("main runtime")
-        .block_on_std(async move {
+        .block_on(async move {
```

Comment:
TIOLI: what do you think about just sticking a `#[tokio::main(basic_scheduler)]` on `main` and turning the whole thing into an `async fn`, instead? that's what I was planning on doing...